### DISCUSSION TITLE
ADDON GIS IN BLENDER HOW INSTALL

### DISCUSSION THREAD

**Discussion by artsantana2021** [domlysz/BlenderGIS#550] [Q&A] [answered]
Bonjour DOMLYSZ,

Excusez moi de vous déranger, je vous écrit depuis Taiwan, en travaillant sur dernier iMac en travaillant dans Blender 2.93.4 et essayant de mettre l’addon GIS Geo. J’ai essayé le link pour télécharger et c’est fait il est dans mon Mac. Puis j’ai essayé de charger l’addon par les preferences de Blender, et je vois le dossier BlenderGIS-225, je double click et il apparaît toute une série des documents, la majorité avec terminaison .py

Mais rien ne marche pour ouvrir l’addon de votre référence. Pouvez vous avoir l’amabilité de suggérer quelque chose pour m’aider (j’espère que oui), et télécharger/INSTALLER le BlenderGisAddon dans l’application Blender?

Autrement pour votre réponse, si vous préférez, mon mail est artsantana@me.com, cordialement

ARTSANTANA

**Comment by domlysz** [comment, marked as answer]
Use the "install" button from the blender preferences dialog and just select the zip file (you don't need to select *.py file inside the zip, just select the archive file itself)

**Reply by artsantana2021** [reply]
Dear ‘help »,

Thank you so much for your precious answer and help. I saw my problem, it was I put the add-on in my computer I didn’t have a ZIP but this:
<MAC file BLENDER-GIS 225>
In other words my Mac automatically opened the zip and transformed in a Mac file. So I couldn’t see any ZIP. Suddenly I thought transform this file in my computer just pressing control button and click in this file, in a menu I could transform it in ZIP, so I got this:
<BlenderGIS-225.zip>
Now it is working very well in my computer, so I am so thankful for your precious help. Hope to send you back this service in future. Have all my best.

Warm regards & best friendly musical thoughts,

         A   R   T   S   A   N   T   A   N   A
